.github/workflows/release.yml

```
@@ -0,0 +1,30 @@
+
+on:
+  push:
+    tags:
+      - "v*"
+
+permissions:
+  contents: write  # to create a Release
+
+jobs:
+  build-ios:
+    uses: ./.github/workflows/ios-unsigned.yml
```

actionlint (1.7.7)

error while parsing reusable workflow "./.github/workflows/ios-unsigned.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)